### PR TITLE
readme: add CoC and contributing sections

### PR DESCRIPTION
**Pull request by tbpg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,5 +45,22 @@ Go Cloud provides generic APIs for:
 -   Server startup and diagnostics: request logging, tracing, and health
     checking
 
+## Contributing
+
+Thank you for your interest in contributing to Go Cloud! :heart:
+
+Everyone is welcome to contribute to Go Cloud, whether it's in the form of code, documentation, bug reports, feature requests, or anything else. We encourage you to experiment with Go Cloud and make contributions to help evolve it to meet your needs!
```
Comment (zombiezen) [inline README.md]: Please wrap lines to 80 columns.

File: README.md [hunk 2]
```diff
@@ -50,5 +45,22 @@ Go Cloud provides generic APIs for:
...
+
+The GitHub repository at [google/go-cloud](https://github.com/google/go-cloud) currently contains [Google Cloud Platform](http://cloud.google.com) and [Amazon Web Services](http://aws.amazon.com) implementations as examples to prove everything is working. Unfortunately, we cannot support every cloud provider directly from the project.
```
Comment (zombiezen) [inline README.md]: Please use reference-style links to keep the prose readable in plain text.

Reference: https://daringfireball.net/projects/markdown/syntax#link

File: README.md [hunk 3]
```diff
@@ -50,5 +45,22 @@ Go Cloud provides generic APIs for:
...
+If you create a repository that implements the Go Cloud interfaces for other cloud providers, let us know!
+We would be happy to link to it here and give you a heads-up before making any breaking changes.
+
+See [the contributing guide](./CONTRIBUTING.md) for more details.
+
+## Code of Conduct
+
+This project is covered by a [Code of Conduct](./CODE_OF_CONDUCT.md).
+
 [travis]: https://travis-ci.com/google/go-cloud
```
Comment (zombiezen) [inline README.md]: Can you move this back up to the section it's supposed to be attached to? It's been gradually falling down to the bottom of the file and it's made me sad.

File: README.md (hunk 1 above)
Comment (tbpg) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (tbpg) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (tbpg) [reply README.md]: Done.

**Pull request merged**